akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardCoordinator.scala

```diff
@@ -210,6 +210,11 @@ object ShardCoordinator {
     }
   }
 
+  /**
+   * Used as a special termination message from [[ClusterSharding]]
+   */
+  private[cluster] case object Terminate extends DeadLetterSuppression
```

Comment:
Becomes public in Java so needs
```
* INTERNAL API 
*/
@InternalApi private[cluster]
```

Reply:
Maybe better then to move this `Terminate` under `private[akka] object Internal { ....`?

Reply:
I moved `Terminate` message under `object Internal` of `ShardCoordinator`, please, review